Source/ProofGeneration/BoogieIsaInterface/IsaProgramGenerator_forAst.cs

```diff
@@ -0,0 +1,425 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Isabelle.Ast;
+using Isabelle.Util;
+using Microsoft.BaseTypes;
+using Microsoft.Boogie;
+using ProofGeneration.BoogieIsaInterface;
+using ProofGeneration.BoogieIsaInterface.VariableTranslation;
+using ProofGeneration.CFGRepresentation;
+using ProofGeneration.ASTRepresentation;
+using ProofGeneration.Util;
+
+namespace ProofGeneration
+{
+  internal class IsaProgramGenerator_forAst
```

Comment:
avoid _ in class names. Here, use `IsaProgramGeneratorForAst` (change file name in the same way)